python/tvm/relay/frontend/pytorch.py

```diff
@@ -4424,7 +4424,10 @@ def _create_typed_const(data, dtype):
     dtype should be a TVM dtype"""
 
     if dtype == "float64":
-        typed_data = _expr.const(np.float64(data), dtype=dtype)
+        if len(data) == 1:
+            typed_data = _expr.const(np.array([np.float64(data)]), dtype=dtype)
+        else:
+            typed_data = _expr.const(np.float64(data), dtype=dtype)
```

Comment:
I read your updates in the issue, thank you for detailed description. Here it looks like a workaround above numpy bug. I think that the best way to fix this issue is to create an issue or a PR in the numpy repo with fix.

Currently, this implementation won't work after fixing the bug in numpy. My suggestion is to create an issue or PR with fix to numpy repository and at the same time change the current implementation in the following way:
```suggestion
        typed_data = np.float64(data)
        if np.isscalar(typed_data):
            typed_data = _expr.const(np.array([typed_data]), dtype=dtype)
        else:
            typed_data = _expr.const(typed_data, dtype=dtype)
```
This code should work also after fixing the bug in Numpy.

Reply:
Thanks! I have submitted a issue to Numpy community [here](https://github.com/numpy/numpy/issues/24661).

Reply:
Thank you! In this case, something like `typed_data = _expr.const(np.asarray(data, dtype="float64"), dtype=dtype)` might be the best solution for this problem.